src/app-layout/index.tsx

```diff
@@ -266,19 +265,33 @@ const OldAppLayout = React.forwardRef(
       }
     );
 
-    const defaultDrawerSize = selectedDrawer?.size || toolsWidth;
+    const getSizes = () => {
+      const sizes: { [id: string]: number } = {};
+      if (!drawers) {
+        return {};
+      }
 
-    const [drawerSize = defaultDrawerSize, setDrawerSize] = useControllable(
-      selectedDrawer?.size,
+      for (const item of drawers.items) {
+        if (item.resizable && item.size) {
```

Comment:
We'll also need to include drawers here that are not `resizable`, because they can still have a size that's different from the default. In the current state the non-resizable drawers always render at a width of `toolsWidth` pixels.

```suggestion
        if (item.size) {
```